src/System.IO.FileSystem.DriveInfo/tests/DriveInfo.Windows.Tests.cs

```diff
@@ -16,24 +16,23 @@ namespace System.IO.FileSystem.DriveInfoTests
 {
     public class DriveInfoWindowsTests
     {
-        [ActiveIssue(27357)]
         [Theory]
-        [InlineData(":", null)]
-        [InlineData("://", null)]
-        [InlineData(@":\", null)]
-        [InlineData(":/", null)]
-        [InlineData(@":\\", null)]
-        [InlineData("Az", null)]
-        [InlineData("1", null)]
-        [InlineData("a1", null)]
-        [InlineData(@"\\share", null)]
-        [InlineData(@"\\", null)]
-        [InlineData("c ", null)]
-        // [InlineData("", "path")] // https://github.com/dotnet/corefx/issues/27269
-        [InlineData(" c", null)]
-        public void Ctor_InvalidPath_ThrowsArgumentException(string driveName, string paramName)
+        [InlineData(":")]
+        [InlineData("://")]
+        [InlineData(@":\")]
+        [InlineData(":/")]
+        [InlineData(@":\\")]
+        [InlineData("Az")]
+        [InlineData("1")]
+        [InlineData("a1")]
+        [InlineData(@"\\share")]
+        [InlineData(@"\\")]
+        [InlineData("c ")]
+        [InlineData("")]
+        [InlineData(" c")]
+        public void Ctor_InvalidPath_ThrowsArgumentException(string driveName)
         {
```

Comment:
Please use AssertExternsions.Throws as it is smart enough to skip the parameters name check when testing UWP.